File: Python/Warmup-1/del_del.py

# this function return a string where "del" has been deleted from the given string if it appears
# otherwise, the function return the string unchange
def del_del(str):
	s = str

	if len(str) >= 4 and str[1:4] == "del":
		s = str[0] + str[4:]

	return s

print(del_del("adelbc"))
print(del_del("adelHello"))
print(del_del("adedbc"))

File: JavaScript/Warmup-1/delDel.js

// given a string, if the string "del" appears starting at index 1, return a string where that "del" has been deleted
// otherwise, return the string unchanged
const delDel = (str) => {

    return (str.length > 4 && str.substring(1,4)) === 'del' ? str[0] + str.substring(4) : str;

}

console.log(delDel('adelbc'));
console.log(delDel('adelHello'));
console.log(delDel('adedbc'));
